src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Configuration/PepSettings.cs

```diff
@@ -9,5 +9,11 @@ public class PepSettings
         /// Gets or sets to disable pep er not
         /// </summary>
         public bool DisablePEP { get; set; }
```

Comment:
Is this the right time to remove disable PEP seeing as we're in here making changes anyways?